os/arch/arm/src/amebasmart/amebasmart_spi.c

```diff
@@ -276,7 +276,7 @@ static struct amebasmart_spidev_s g_spi1dev = {
 	.spi_mosi = PB_28,
 	.spi_miso = PB_27,
 	.spi_sclk = PB_25,
-	.spi_cs0 = PB_31,
+	.spi_cs0 = PB_26,
```

Comment:
https://github.com/Samsung/TizenRT/commit/f90a81026a213f45b239fde173ec9e30367297dc
This is modified already. Let's revert this from this commit.